app/i18n/zh-cn/sub.php

```diff
@@ -32,19 +32,25 @@
 		'description' => '描述',
 		'empty' => '此源为空。请确认它是否正常更新。',
 		'error' => '此源遇到一些问题。请在确认是否能正常访问后重试。',
-		'in_main_stream' => '在首页中显示',
 		'informations' => '信息',
 		'keep_history' => '至少保存的文章数',
 		'moved_category_deleted' => '删除分类时，其中的 RSS 源会自动归类到 <em>%s</em>',
+		'mute' => 'mute', // TODO
 		'no_selected' => '未选择 RSS 源。',
 		'number_entries' => '%d 篇文章',
+		'priority' => array(
```

Comment:
I do not know whether it is better, just a suggestion: we could have a `'_' => 'Visibility'` (or `'Priority'`) instead of the standalone key a bit lower (just to keep things grouped)

Reply:
Sure, why not

Reply:
Done